### PR TITLE
put login button to navbar

### PR DESCRIPTION
**Pull request by emmyachieng**

### CODE REVIEW

**Comment by TraceyOnim** [comment]
The button can be well-positioned in the navbar

image

Also, I'm not able to see the different options for logging in

**Comment by emmyachieng** [comment]
A user can toggle between the sign in and logout button in the navbar, and the play game button is only displayed when one has logged in.

**Comment by emmyachieng** [comment]
A user is able to toggle betweeen the sign in and logout button, and the play game button is displayed only when the user has logged in.

**Comment by emmyachieng** [comment]
- A user is able to toggle between the sign in and logout button.
- The play game button only appears when a user is signed in.

Peek 2

File: lib/quadquizaminos_web/templates/layout/navbar.html.leex [hunk 1]
```diff
@@ -17,6 +17,9 @@
         <%= if Quadquizaminos.Accounts.user_has_role?(@current_user, ["admin"]) do %>
           <li><%= admin_dropdown() %></li>
         <% end %>
+        <% else %>
+         <%= hide_or_show_sign_up_button() %>
+
```
Comment (WanjikuMac) [inline lib/quadquizaminos_web/templates/layout/navbar.html.leex:22]: The admin button is after the logout button, the order should be `admim` `logout` button not `logout` `admin` @emmyachieng you can move this `<%= link "Logout", to: Routes.auth_path(QuadquizaminosWeb.Endpoint, :delete), method: "delete", class: "button logout"%>` to line 19  after the end tag

Comment (emmyachieng) [reply lib/quadquizaminos_web/templates/layout/navbar.html.leex:22]: okay

**Pull request merged**